<?xml version="1.0" encoding="utf-8"?>
<testsuites>
	<testsuite name="pytest" errors="0" failures="1" skipped="0" tests="2" time="1.188" timestamp="2022-05-30T14:49:18.865623" hostname="PC">
		<testcase classname="tests.test_me" name="test_01" file="tests\test_me.py" line="0" time="0.025"/>
		<testcase classname="tests.test_me" name="test_02" file="tests\test_me.py" line="4" time="0.014">
			<failure message="def test_02():&#10;&gt;       assert False&#10;E       assert False&#10;&#10;&#10;tests\test_me.py:6: AssertionError" type="assert False"/>
		</testcase>
	</testsuite>
</testsuites>
